src/core/Snackbar/__tests__/Snackbar.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import { LitmusThemeProvider } from "../../../theme";
+import { Snackbar } from "../Snackbar";
+
+beforeEach(() => {
+  render(
+    <LitmusThemeProvider>
+      <Snackbar message={"Test Snackbar"} open={true} />
+    </LitmusThemeProvider>
+  );
+});
+
+test("Check for inheritance of children props", () => {
+  // Matching a string:
+  screen.getByText("Test Snackbar"); // full string match
+  screen.getByText("est Snack", { exact: false }); // substring match
+  screen.getByText("tEsT sNaCkBaR", { exact: false }); // ignore case
+
+  // Matching a regex:
+  screen.getByText(/Snackbar/); // substring match
+  screen.getByText(/sNaCkBaR/i); // substring match, ignore case
```

Comment:
these as well...We should be only checking full string in the exact case as it is, right?